### DISCUSSION TITLE
Repeat Label

### DISCUSSION THREAD

**Discussion by khaliduzzaman-mredul** [SkalskiP/make-sense#362] [General]
Is there any possible way in Makesense.ai to repeat the previous label to the next one as we can do in Roboflow?

**Comment by garminTestPilot50** [comment]
Often I try to use the AI just to get the bounding box, but it gets the wrong label.  Ideally it would be great to also be able to say "use the last label" for the same bounding box order.  It tends to get the boxes in the right order, but there's no good way to batch across many frames other than saving out all the label text files and modifying them after the fact with python
